chainer/functions/array/resize_images.py

```diff
@@ -25,38 +188,30 @@ def forward(self, inputs):
         x, = inputs
         xp = backend.get_array_module(x)
 
-        B, C, H, W = x.shape
+        _, C, H, W = x.shape
+        out_H, out_W = self.out_H, self.out_W
+
+        # Compute indices and weights.
+        v = xp.linspace(0, H - 1, num=out_H, dtype=numpy.float)
+        u = xp.linspace(0, W - 1, num=out_W, dtype=numpy.float)
+        vw, v = xp.modf(v)
+        uw, u = xp.modf(u)
+        v = v.astype(numpy.intp)
+        u = u.astype(numpy.intp)
+        vw = vw.astype(x.dtype)
+        uw = uw.astype(x.dtype)
+
+        # Meshgrid-like operation. Meshgrid can cause
+        # performance loss due to memory consumption.
+        v = xp.broadcast_to(v[:, None], (out_H, out_W))
```

Comment:
Old NumPy does not have `broadcast_to` method.

Reply:
```suggestion
        # xp: numpy, cupy, or chainer.functions
        if hasattr(xp, 'broadcast_to'):
            v = xp.broadcast_to(v[:, None], (out_H, out_W))
        else:
            # numpy 1.9 doesn't support broadcast_to method
            dummy = xp.empty((out_H, out_W))
            v, _ = xp.broadcast_arrays(v, dummy)
```

@grafi-tt @okuta How about this?

Reply:
It's actually from this part, so I think it's better to move this function somewhere more proper and call it from here.
https://github.com/chainer/chainer/blob/f83790cfb97191491a1f26cca1c5fba4166fb2c2/chainer/functions/normalization/layer_normalization.py#L7-L15